reference/cli/sailsdebug.md

```diff
@@ -38,9 +38,11 @@ Now you can lift your Sails app in debug mode
 $ sails debug
 ```
 
-One the application is launched, visit the page http://127.0.0.1:8080?port=5858. Now you can request your app as usally on port 1337 and debug your code from the browser.
+Once the application is launched, visit http://127.0.0.1:8080?port=5858. Now you can request your app as usual on port 1337 and debug your code from the browser.
 
-> If you don't see your files in the browser at http://127.0.0.1:8080?port=5858 or if it's very slow to load, try to run node-inspector with --no-preload argument see [node-inspector](https://github.com/node-inspector/node-inspector) for more details
+@TODO Add explanations about the role of the differents components
```

Comment:
Aahha, I don't know if the `todo` tag will be hidden. 
It's my bad, I wanted to add a paragraph but I did't have the time.
I add it right away